src/main/kotlin/net/maizegenetics/phgv2/cli/CreateFastaFromHvcf.kt

```diff
@@ -118,6 +118,8 @@ class CreateFastaFromHvcf : CliktCommand( help = "Create a fasta file from a hvc
 
     /**
      * Function to create haplotype Sequences for each of the haplotype variants in the hvcf
+     * TODO LCJ - determine how sampleName is used here.
```

Comment:
Rewrite the debug comment to specify how we are using the sample name in the new code.